### ISSUE TITLE
SDcard not showing the correct size in Windows or on Switch

### ISSUE THREAD

**Issue by boomerams4l** [lulle2007200/SwitchSDSetup#4]
I used the latest script (V0.2.11) to try out Android and Ubuntu along side my hos_data partition and everything worked fine the first go around, but when i decided to ditch Android for just a hos_data partition alongside Ubuntu my Linux VM with Ubuntu 19.10 shows the two separate partitions sized correctly, but when connected to windows or booted up in the switch the size still shows the full size of the actual sdcard (1tb). Everything still works fine but it is just a little annoying not knowing how much actual free space i have left. Some pictures are attached and any help is appreciated.

![IMG_20200427_135645](https://user-images.githubusercontent.com/60925156/80405083-21165000-8890-11ea-9e69-59376a43d18f.jpg)

![IMG_20200427_135748~2](https://user-images.githubusercontent.com/60925156/80405087-22e01380-8890-11ea-8af8-9557b7ec3160.jpg)

![IMG_20200427_135959](https://user-images.githubusercontent.com/60925156/80405090-2378aa00-8890-11ea-8a0c-b001a25a4e37.jpg)

**Comment by lulle2007200** [comment]
Thats odd. Can you run `sudo sgdisk -O /dev/sdx` and `sudo sgdisk -p /dev/sdx` and send the output?
Seems like mbr table is broken. 
Try `sudo setup.sh --fix-mbr` after doing the above.

**Comment by boomerams4l** [comment]
Sorry for taking so long to get back to this issue, but it has been a long couple of days.

While initially waiting for a reply to my issue I was able to fix the problem by messing around in DiskGenius, although, I don't remember exactly what I did, off the top of my head, it was definitely a problem with with the mbr table. I tried reformatting and everything else, but it wasn't until I started to mess with the mbr that I was finally able to get things to show up correctly.

I do have a friend who wants me to setup his switch with Android and Ubuntu once his new sdcard comes in (Friday or Saturday) so if you would like to keep this issue open until then I can report back if I run into any similar issues, otherwise, we can probably consider this issue closed. 

Appreciate the help either way.
